moveit_simple/src/moveit_simple.cpp

```diff
@@ -220,8 +220,102 @@ bool Robot::getJointSolution(const Eigen::Affine3d &pose, double timeout,
 }
 
 
+
+bool Robot::getJointSolution(const Eigen::Affine3d &pose, const std::string& custom_tool_frame,
+                             double timeout, const std::vector<double> & seed,
+                             std::vector<double> & joint_point) const
+{
+  bool GET_JOINTS_FLAG = false;
```

Comment:
Variable names should be `lower_case`.  Don't append the word `flag`.   It's redundant with the data type, which the IDE displays for you.

Reply:
Oh yeah, I have the overloads ready but the tests have been commented out due to another issue somewhere.. I'll just send them out shortly.